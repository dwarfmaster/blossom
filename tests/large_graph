16 19
0 1
1 4
4 3
3 2
2 13
13 0
1 15
4 5
5 3
2 6
6 7
7 8
8 12
12 10
10 9
9 14
14 7
8 11
11 12
