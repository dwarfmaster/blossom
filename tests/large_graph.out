3 -- 2
13 -- 0
1 -- 15
4 -- 5
6 -- 7
12 -- 10
9 -- 14
8 -- 11
